tests/integration/states/test_ssh_known_hosts.py

```diff
@@ -54,8 +59,7 @@ def test_present(self):
                 )
                 self.skipTest('Unable to receive remote host key')
             except AssertionError:
-                # raise initial assertion error
-                raise err
+                six.reraise(*sys.exc_info())
```

Comment:
The original code raises the first exception that is the `err`. The new code reraises the last (nested) AssertionError. Is is expected?

Reply:
@s0undt3ch this one still need to be fixed here. It's minor, but it directly relates.